Lessson16/script.js

```diff
@@ -229,22 +319,60 @@ getDayOfWeek(10);
      below 60 -> "F"
    - Log the grade.
 */
+function gradeChecker(score) {
+  if (score >= 90) {
+    console.log('A');
```

Comment:
A better log message could be something like: `console.log("Your grade is: A");`